packages/firebase_dynamic_links/lib/src/short_dynamic_link.dart

```diff
@@ -0,0 +1,16 @@
+// Copyright 2017 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+part of firebase_dynamic_links;
+
+/// Response from creating a short dynamic link with [DynamicLinkComponents].
+class ShortDynamicLink {
+  ShortDynamicLink._(this.shortUrl, this.warnings);
+
+  /// Short url value.
+  final Uri shortUrl;
+
+  /// Gets information about potential warnings on link creation.
```

Comment:
Getters should be documented with a noun phrase describing what the property is.

https://www.dartlang.org/guides/language/effective-dart/documentation#prefer-starting-variable-getter-or-setter-comments-with-noun-phrases